op-node/p2p/gossip.go

```diff
@@ -380,15 +382,21 @@ func BuildBlocksValidator(log log.Logger, cfg *rollup.Config, runCfg GossipRunti
 		}
 
 		if blockVersion.HasBlobProperties() {
-			// [REJECT] if the block is on a topic >= V3 and has a blob gas used value that is not zero
-			if payload.BlobGasUsed == nil || *payload.BlobGasUsed != 0 {
-				log.Warn("payload is on v3 topic, but has non-zero blob gas used", "bad_hash", payload.BlockHash.String(), "blob_gas_used", payload.BlobGasUsed)
+			// [REJECT] if the block is on a topic >= V3 and has a nil blob gas used
+			if payload.BlobGasUsed == nil {
+				log.Warn("payload is on v3 topic, but has nil blob gas used", "bad_hash", payload.BlockHash.String())
+				return pubsub.ValidationReject
+				// [REJECT] if the block is on a topic >= V3 and has a non-zero blob gas used field pre-Jovian
+			} else if !cfg.IsDAFootprintBlockLimit(uint64(payload.Timestamp)) && *payload.BlobGasUsed != 0 {
```

Comment:
Should this be updated to be `IsJovian` given that its scheduled for jovian?

Reply:
We will remove all of the feature flags in a single PR soon.